console/console-init/ui/src/modules/msg-and-iot/dailogs/components/IoTReviewStep/IoTReviewStep.tsx

```diff
@@ -4,15 +4,14 @@
  */
 
 import React from "react";
-import { isIoTProjectValid } from "modules/msg-and-iot/dailogs";
-import { IIoTProjectInput } from "modules/msg-and-iot/dailogs";
-import { IoTReview } from "../IoTReview";
+import {
```

Comment:
folder structure consistency is broken. dialogs folder don't need further components,containers etc folder. we have folder name s component name as we are following in other modules. 
I think these are not dialog components. These should move under normal component and containers folder.
All dialogs are import in RootModal.tsx and will open by using dispatch .